front/src/applications/operationalStudies/components/SimulationResults/ChartHelpers/drawElectricalProfile.jsx

```diff
@@ -74,12 +75,15 @@ const drawElectricalProfile = (
     if ((rotate && height < -24) || (!rotate && width > 60)) {
       const textZone = chart.drawZone.select(`#${groupID}`);
 
+      // get the width of the text element and adding a few pixels for readability
+      const svgWidth = getElementWidth(dataSimulation.text, 8, `#${groupID}`) + 10;
```

Comment:
Add a constant here to improve the readability
```suggestion
      const labelFontSize = 8; // in px
      const svgWidth = getElementWidth(dataSimulation.text, labelFontSize, `#${groupID}`) + 10;
```

Reply:
done :)